src/Simulation/Simulators/QuantumSimulator/StateDumper.cs

```diff
@@ -2,7 +2,13 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Collections.Generic;   // IComparer
```

Comment:
This is based on `main`. I don't know any details about the .NET 6 refactoring (anything related to .NET 6 is a black box for me at the moment).
I'm removing the comments in these `using` lines to preserve the code style (I was planning to do that any way). But I would like to refrain right now from proceeding to `global using` since that change is unrelated to dump and may be distracting.

Reply:
Makes sense. By the time the third PR in your plan above hits, `main` will likely be on .NET 6 so maybe this will fit best as part of that cleanup?